### PR TITLE
Fixes Navigation Bar Bug

### LINKED ISSUE #153: Fix top header on celo.org
if you go to celo.org and then the about page and then press the celo logo to go back to homepage the top nav is on top of the blue banner. I

It should be below it

### PR DESCRIPTION
**Pull request by aaronmgdr**
# changes

fixes issue with top nav being on top of blue banner when navigating from another page to home page. This was due to 
1. toggleing the state rather than setting it based on announcement visibility
2. using componentDidUpdate rather than an explict useLayoutEffect with route as a dep

also make it a little quicker 

# issue 
fixes #153 

Screen Shot 2021-04-08 at 5 14 08 PM

### CODE REVIEW

File: src/header/BlueBanner.tsx [hunk 1]
```diff
@@ -11,36 +12,37 @@ interface Props {
...
-    this.ref.current.measure((_x, _y, _w, height) => {
-      this.props.getRealHeight(height)
-    })
-  }
-  render() {
+export function BlueBanner(props: Props) {
+    const ref = React.useRef<View>()
+    const router = useRouter()
+
+    React.useLayoutEffect(() => {
```
Comment (Henrynunez112) [inline src/header/BlueBanner.tsx:19]: is React.useLayoutEffect the same as useEffect??

File: src/header/Navigation.tsx [hunk 2]
```diff
@@ -129,7 +125,7 @@ function useMobileMenu(): [boolean, () => void] {
 
  export default function Header() {
     const {bannerHeight} = useScreenSize()
-    const {isBannerShowing, toggleBanner} = useBanner()
+    const {setBannerVisible, isBannerShowing} = useBanner()
     const {pathname} = useRouter()
```
Comment (Henrynunez112) [inline src/header/Navigation.tsx:129]: why the name change?

**Review by Henrynunez112 — APPROVED** [review]
Good to go! 😎

File: src/header/Navigation.tsx (hunk 2 above)
Comment (aaronmgdr) [reply src/header/Navigation.tsx:129]: toggle implies each time its called it just flips back and forth. on and off

but now it doesnt it sets it to the value of the param

File: src/header/BlueBanner.tsx (hunk 1 above)
Comment (aaronmgdr) [reply src/header/BlueBanner.tsx:19]: LayoutEffect runs after it has rendered to the DOM. good for measuring things.

**Pull request merged**